indico/modules/events/abstracts/templates/abstract/public.html

```diff
@@ -335,9 +345,54 @@ <h3 class="abstract-title">
                     <div class="i-timeline-item-box header-indicator">
                         <div class="i-box-header flex">
                             <div class="stretch">
-                                {% trans name=abstract.judge.name, action=abstract.state.title|lower, date=abstract.judgment_dt|format_human_date -%}
-                                    <strong>{{ name }}</strong> {{ action }} this abstract on {{ date }}.
-                                {%- endtrans %}
+                                {% set judgement_dt = abstract.judgment_dt|format_human_date %}
+                                {% set action = abstract.state.name %}
+                                {% if abstract.can_judge(session.user) %}
+                                    {% set judge_name = abstract.judge.name %}
+                                    {% if action == 'withdrawn' %}
+                                        {% trans name=judge_name, date=judgement_dt -%}
+                                            <strong>{{ name }}</strong> withdrew this abstract on {{ date }}.
```

Comment:
"on yesterday" once again ;) maybe "withdrew this abstract ({{ date }})" instead?